metadata/updater/updater.go

```diff
@@ -662,6 +662,10 @@ func (update *Updater) GetTrustedMetadataSet() trustedmetadata.TrustedMetadata {
 	return *update.trusted
 }
 
+func (update *Updater) SetRefTime(t time.Time) {
```

Comment:
```suggestion
func (update *Updater) UnsafeSetRefTime(t time.Time) {
```

Reply:
We have some other `Unsafe` methods like [this](https://github.com/theupdateframework/go-tuf/blob/41866142c66ab85ba57bc06fe01be1685fc2afd9/metadata/config/config.go#L46)

If would suggest prefixing this with Unsafe too, and add a comment on why this is needed, and the consequences of using it.